### DISCUSSION TITLE
Does anyone have a better description of how the <segment:text,value,value> functionality works?

### DISCUSSION THREAD

**Discussion by ataylorm** [Stability-AI/StableSwarmUI#135] [Q&A] [answered]
Title says it all, seems like it could be useful, but not sure how to get it working?

**Comment by mcmonkey4eva** [comment, marked as answer]
It's explained in the feature announcement here: https://github.com/Stability-AI/StableSwarmUI/discussions/11#discussioncomment-7236821

**Reply by ataylorm** [reply]
Thank you so much!  That is awesome now that I know how to use it.
